crates/edr_eth/src/remote/client.rs

```diff
@@ -89,10 +86,10 @@ pub enum RpcClientError {
     },
 
     /// A response is missing from a batch request.
-    #[error("Missing response for method: '{method_invocation:?}' for request id: '{id:?}' in batch request")]
+    #[error("Missing response for method: '{method:?}' for request id: '{id:?}' in batch request")]
     MissingResponse {
         /// The method invocation for which the response is missing.
-        method_invocation: MethodInvocation,
+        method: Box<RequestMethod>,
```

Comment:
I had to box this as clippy was complaining about a `Result<SerializedRequest, RpcClientError>` being 152 bytes large which is too big. The weird thing is that I checked, and before this PR, its size was 408 bytes.